### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mingyli**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
```
Comment (RahulTirumalareddy) [inline .github/ISSUE_TEMPLATE/bug_report.md]: nit: iPhone 6

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ - OS: [e.g. iOS8.1]
```
Comment (RahulTirumalareddy) [inline .github/ISSUE_TEMPLATE/bug_report.md]: nit: iOS 8.1

**Review by RahulTirumalareddy — CHANGES_REQUESTED** [review]
Great addition, just a couple of spelling nits!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mingyli) [reply .github/ISSUE_TEMPLATE/bug_report.md]: 👍

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mingyli) [reply .github/ISSUE_TEMPLATE/bug_report.md]: 👍

**Review by RahulTirumalareddy — APPROVED** [review]
ballin' as usual

**Pull request merged**